docs/docs/data/create.tsx

```diff
@@ -350,7 +350,135 @@ const createEndpointTableData = [
+    "description": (
+        <>
+          List (comma-separated) of features to disable in a particular meeting. (added 2.5)
+          <br /><br />
+          Available options to disable:
+          <ul>
+            <li>
+              <code className="language-plaintext highlighter-rouge">
+                breakoutRooms
+              </code> - <b>Breakout Rooms</b>
```

Comment:
Please also document in https://github.com/bigbluebutton/bigbluebutton/blob/v3.0.x-release/docs/docs/development/api.md?plain=1#L114